live-examples/html-examples/text-content/main.html

```diff
@@ -1,5 +1,5 @@
 <header>
-    <img class="logo" src="/media/examples/lizard.png" />
+    <img class="logo" src="/media/cc0-images/Gecko--320x213.jpg" />
```

Comment:
This image is sized really small. I know this is not introduced by this PR but, perhaps you can also go ahead and change the size from:

```
header > .logo {
    height: 2.5rem;
...
```

to

```
header > .logo {
    height: 4.5rem;
...
```